datalad/core/local/create.py

```diff
@@ -459,7 +459,8 @@ def __call__(
 
         if _seed is None:
             # just the standard way
-            uuid_id = uuid.uuid1().urn.split(':')[-1]
+            # use a fully random identifier (i.e. UUID version 4)
+            uuid_id = str(uuid.uuid4())
         else:
             # Let's generate preseeded ones
             uuid_id = str(uuid.UUID(int=random.getrandbits(128)))
```

Comment:
why not to just always use this one (so no `if _seed` conditioning) then so we have a single implementation instead of conditioning here on `_seed`:
```
def uuid4():
    """Generate a random UUID."""
    return UUID(bytes=os.urandom(16), version=4)
```
so the only difference would be from stock uuid4 is that we would use  `random.getrandbits` and not  `os.urandom` ("Return a bytes object containing random bytes suitable for cryptographic use.").  Not sure if "cryptographic" aspect would really matter here...?

Reply:
may be just add an explicit `version=4` as well

Reply:
This is part of an effort towards a certified identifier setup. Being able to say "this is a UUID4" is better than "this should not be too different from a UUID4".

That being said, I am also OK with leaving things as they are and adding an explicit, separate mode where the properties of all identifiers are precisely defined.

Reply:
nah, with the above I was just trying to reduce number of "modes" of operation in the code, not to increase it... I am ok with this PR as is then.